pkg/golinters/goanalysis/runners.go

```diff
@@ -118,6 +126,13 @@ func buildIssues(diags []Diagnostic, linterNameBuilder func(diag *Diagnostic) st
 	return issues
 }
 
+func formatSuggestFix(fix string) string {
+	if fix == "" {
+		return ""
+	}
+	return fmt.Sprintf("\n```\n%s\n```\n", fix)
```

Comment:
```suggestion
	return fmt.Sprintf("\n```suggestion\n%s\n```\n", fix)
```

Reply:
it is needed to properly format a suggestion in the comments created by the action

Reply:
Done.